### PR TITLE
Epic/goal management

### PR DESCRIPTION
**Pull request by Tshimangadzo**

### CODE REVIEW

**Review by Sinovuyo97 — APPROVED** [review]

**Review by Itumeleng-Koalane — APPROVED** [review]

File: G4L.UserManagement.API/Controllers/GoalController.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        public async Task<IActionResult> GetAllGoals()
+        {
+            var goalList = await _goalService.GetAllGoalsAsync();
+            return Ok(goalList);
+        }
+
+        [HttpGet]
+        [Route("goalId")]
+        public async Task<IActionResult> GetGoal([FromRoute] Guid goalId)
+        {
```
Comment (Thapelo-Seema) [inline G4L.UserManagement.API/Controllers/GoalController.cs:44]: Is this method supposed to get the goals of a specific user ? if so, why are we not using the id we are getting from the route?

**Comment by Tshimangadzo** [comment]
Hi, @Thapelo-Seema 

I can see you have made the comment, however, I can't retrieve the comment, did you delete it?

If that is a requested change, please close the PR after, so that changes can be made, and I will reopen pull request once the issues have been resolved.

Please let me know if you deleted the comment.

**Review by nhlamulo-maluleka — COMMENTED** [review]
Before continuing merging this branch with SIT, can we get the `Task/fix-backend` branch working and merge that with SIT?

**Comment by nhlamulo-maluleka** [comment]
This is because the changes that we have on the `Task/fix-backend` are different from the ones on the `Epic/goal-management`. Or we can just merge `Task/fix-backend` with `Epic/goal-management` before merging it on with the `SIT_branch`

**Pull request closed without merging**